packages/compass-preferences-model/src/preferences.ts

```diff
@@ -558,6 +559,22 @@ const modelPreferencesProps: Required<{
       long: 'Register Compass as a handler for mongodb:// and mongodb+srv:// URLs',
     },
   },
+  /**
+   * Determines if the toggle to edit connection string for new connections
+   * should be in the off state or in the on state by default
+   */
+  protectConnectionStringsForNewConnections: {
+    type: 'boolean',
+    required: false,
+    default: false,
+    ui: true,
+    cli: true,
+    global: true,
+    description: {
+      short:
+        'Sets the default state of "Edit connection string" toggle for new connections',
```

Comment:
It sets the inverse of it, right? 

```suggestion
        'If true, "Edit connection string" is disabled for new connections by default',
```

Reply:
That is right, thanks for the copy :) I will accept it here, as it is.